lib/remoteResource.js

```diff
@@ -108,7 +145,8 @@ const deleteRemoteResources = async (resources) => {
   } catch (error) {
     log.error(error);
   }
-};
+
+}
 
 const getRemoteResources = async (clusterId) => {
```

Comment:
on line https://github.com/razee-io/ClusterSubscription/blob/naod/issue126/lib/remoteResource.js#L155 need to update KIND to be 'RemoteResource'

Reply:
and API_VERSION becomes RR_API_VERSION